Documentation/operations/upgrade.rst

```diff
@@ -325,6 +325,9 @@ Annotations:
   default only matters if no explicit selection of the subnet occurs, i.e.
   specifying subnet IDs or tags still takes precedence.
 
+* NodeExternalIP of the local node is now correctly included into the host
+  entity (it used to belong to the world entity).
```

Comment:
I wonder if what "the host entity" refers to will be clear in this context? Maybe we could make it:
```
into the ``host`` entity for network policies
```
or link to [the Entity section](https://docs.cilium.io/en/stable/policy/language/#entities-based). WDYT?